linera-base/src/identifiers.rs

```diff
@@ -167,27 +167,6 @@ pub enum BlobType {
     Data,
 }
 
-impl Display for BlobType {
-    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
-        match self {
-            BlobType::Data => write!(f, "Data")?,
-        };
-
-        Ok(())
-    }
-}
-
-impl FromStr for BlobType {
-    type Err = anyhow::Error;
-
-    fn from_str(s: &str) -> Result<Self, Self::Err> {
-        match s {
-            "Data" => Ok(BlobType::Data),
-            _ => Err(anyhow!("Invalid blob type: {}", s)),
-        }
-    }
-}
-
```

Comment:
I would rather keep these implementations and use `serde_json` in here instead, that way we can also change a lot less code

Reply:
Done https://github.com/linera-io/linera-protocol/pull/2364/commits/81bc43cf5cb39ac863f0c840949f8cd42ebc00b4